Sources/SwiftInspectorVisitors/TypeDescription.swift

```diff
@@ -72,23 +78,27 @@ public enum TypeDescription: Codable, Equatable {
    * We’ll Get There™
    */
 
-  /// A source-code representation of this type that can be used in code generation.
+  /// A canonical representation of this type that can be used as source code.
   public var asSource: String {
     switch self {
-    case let .simple(name):
-      return name
-    case let .array(type):
```

Comment:
I can see a world where maybe one day we bring this back but I like the idea of keeping the API simple until we have a clear need.